redis-snapshot-replicator/main.tf

```diff
@@ -0,0 +1,273 @@
+resource "aws_iam_role" "replication-role" {
+  name = "${var.name}-replication-role"
+
+  assume_role_policy = <<POLICY
```

Comment:
Sorry for being a pain in the a**, but could you use [`aws_iam_policy_document`](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html) data resources to specify policies? 😅